opentech/apply/projects/templatetags/contract_tools.py

```diff
@@ -10,6 +10,9 @@ def user_can_upload_contract(project, user):
     if user.is_apply_staff:
         return project.status != COMMITTED
 
+    if project.status != CONTRACTING:
```

Comment:
Why did you add this? If there is a contract to sign they should be able to upload, which the following logic covers.

Reply:
Poor assumption on my part, we've gated access to other things after moving out of a relevant state and I thought that applied here too since the spec didn't call it out one way or another.

Removed and updated tests to reflect the correct flow.